File: trunk/java/src/game/Die.java
package game;

public class Die
{
	private char letter;
	private boolean marked;
	
	public Die(char letter)
	{
		this.letter = letter;
		this.marked = false;
	}
	
	public char getLetter()
	{
		return this.letter;
	}
	public void setLetter(char letter)
	{
		this.letter = letter;
	}
	public boolean isMarked()
	{
		return this.marked;
	}
	public void setMarked(boolean marked)
	{
		this.marked = marked;
	}
	
	@Override
	public String toString()
	{
		return Character.toString(this.letter);
	}
}

File: csharp/Boggle/Die.cs
class Die
{
    public bool marked { get; set; }
    public char letter { get; set; }

    public Die(char letter)
    {
        this.letter = letter;
        this.marked = false;
    }

    public override string ToString()
    {
        return this.letter + "";
    }
}

